bot/resources/tags/regex.md

```diff
@@ -0,0 +1,15 @@
+**Regular expressions**
+Sometimes you might want to detect a certain pattern of characters in a string without looking for every possible substring. Regular expressions (regex) are a language for specifying patterns that is used in many programming languages. Python's standard library includes the `re` module, which defines functions for using regex patterns.
+
+**Example**
+We can use regex to pull out all the numbers in a sentence:
+```py
+>>> import re
+>>> x = "On Oct 18, 1963 Félicette the cat was launched into space aboard Veronique AGI sounding rocket No. 47."
```

Comment:
What about

> "On Oct 18, 1963 the cat was launched into space on rocket No. 47."

The sentence doesn't need to be very long to get the point about digits across.

Reply:
Shortened it even more to fit on one line in the embed